### PR TITLE
Main page frontend

### PR DESCRIPTION
**Pull request by BohdanOne**
I think this is almost all i could do until we have backend running. 
I've commented places where code should be replaced when we have server side logic done.
In header there is hidden hamburger button which we can use if we implement some kind of user panel.
I'm not sure if I'm using Styled Components correctly so any feedback on this will be very much welcomed.
React Router is implemented in sorting selects, but now i think it's relevant and I'm going to change that part tomorrow.
Layout is focused on smaller devices, but on laptop should still looks ok. I would also like to try implement swipes on Slider Component.

### CODE REVIEW

**Comment by BohdanOne** [comment]
I've done some refactoring. At that moment, the only thing I would like to add here is to handle swipe on movie slider.

File: client/src/components/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+const Heading = styled.header`
+  align-items: center;
+  display: flex;
+  height: 10vh;
+  justify-content: center;
+  width: 100%;
+  `;
+
+const Title = styled.h1`
```
Comment (DominikKrajcer) [inline client/src/components/Header.jsx]: It's better to store styles in a separate folder e.g. if you have Header.jsx component you can create HeaderStyles.jsx with styled components/css.

File: client/src/components/main-page/slider/Slider.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      translateValue: prevState.translateValue - 100
+    }));
+  }
+
+  render() {
+    return (
+      <Slides>
+
+        <SlideContainer translate={this.state.translateValue}>
+          { this.props.images.map((image, i) => <Slide key={i} image={image} onMovieSelect={ this.props.onMovieSelect }/>)}
```
Comment (DominikKrajcer) [inline client/src/components/main-page/slider/Slider.jsx]: If you have not a fixed list, using index as an ID is an anti-pattern. Try to create some counter or use a lib which provide functions to generate uuid.

File: client/src/components/main-page/SelectBar.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    color: ${props => props.theme.colors.ternary}
+  }
+`;
+
+const SelectBar = props => {
+  return (
+      <Select>
+        <SelectOption
+          id="selectPopular"
+          className={props.active === 'popular' ? 'active' : null}
```
Comment (DominikKrajcer) [inline client/src/components/main-page/SelectBar.jsx]: Magic numbers.

File: client/src/components/main-page/SelectBar.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          onClick={() => {
+            props.onSortSelect('popular');
+          }}
+        >
+          most popular
+        </SelectOption>
+        
+        <SelectOption
+          id="selectBestRated"
+          className={props.active === 'bestRated' ? 'active' : null}
```
Comment (DominikKrajcer) [inline client/src/components/main-page/SelectBar.jsx]: Magic num.

File: client/src/components/main-page/SelectBar.jsx (hunk 3 above)
Comment (BohdanOne) [reply client/src/components/main-page/SelectBar.jsx]: could you suggest how to write that properly? Should i just assign those values to variables and use variables instead?

Comment (DominikKrajcer) [reply client/src/components/main-page/SelectBar.jsx]: Yes, you should. Btw, in more complex conditional even better solution is to create a function.

**Pull request closed without merging**